lang/gcc14/Portfile

```diff
@@ -69,6 +68,17 @@ depends_lib-append  port:cctools \
 depends_run-append  port:gcc_select \
                     path:share/doc/libgcc/README:libgcc
 
+platform darwin {
```

Comment:
Is this line actually unnecessary as the linter says?

Reply:
I believe that is the default platform -- but I would just leave that alone. You don't want to get into a rehash of how the gcc portfiles are written in general, as that will take us down a very deep rabbit hole that we may never escape with others involved in it.

Reply:
Lint is not complaining about `platform` but rather about `platforms`. It's also a [false positive](https://trac.macports.org/ticket/70997) in this case.